cmd/fyne/version_test.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParsePkgString(t *testing.T) {
+	type test struct {
+		in string // Test input
+		es string // Expected output
```

Comment:
Can you use meaningful variable names? If it was "expectedOutput" then you wouldn't need the comment...

Reply:
So, the Fyne codebase doesn't follow Go stdlib conventions?

- https://github.com/golang/go/wiki/CodeReviewComments#variable-names
- https://talks.golang.org/2014/names.slide#6
- and, of course, [Rob Pike's comments](http://doc.cat-v.org/bell_labs/pikestyle) on the matter.

Reply:
I would not go so far as to say Fyne does not follow Go conventions, but I would say that the project follows the belief that if you have to have inline comments then your code could be clearer. Of course "i" is an iterator and "b" could be a buffer - where the context is clear and a convention is followed. I do not think that "es" is common shorthand for "expected output" - looking at slide#8 It would seem to indicate that "expected" could be completely inline with their recommendation.